zerver/webhooks/buildbot/doc.md

```diff
@@ -0,0 +1,11 @@
+Zulip supports integration with CircleCI and can notify you of
+your build statuses.
+
+1. {!create-stream.md!}
+
+2. {!create-bot-construct-url-indented.md!}
+
+3. Add a new Zulip reporter in your configuration file with the endpoint set to your
+ zulip server and token set to the bot's api key
+
+{!congrats.md!}
```

Comment:
I would recommend adding a sample screenshot of a sample Buildbot message at the end here. Make sure the bot has the name `Buildbot Notifications` and that the stream is named `buildbot`. Also, the bot used should have the Buildbot logo as its avatar. :)